src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,159 @@
+package duke;
+
+import duke.Deadlines;
+import duke.Events;
+import duke.Task;
+import duke.ToDos;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void showWelcomeMessage() {
+        System.out.println("Hello! I'm duke.Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void printList(Task[] list, int size) {
+        int position = 1;
+        System.out.println("    Here are the tasks in your list:");
+        for( int i = 0 ; i < size ; i ++ ) {
+            System.out.println( "    " + position  + "." + list[i]);
+            position ++;
+        }
+    }
+
+    public static void markTaskDone(Task[] list , String word) {
```

Comment:
Suggest to change the variable name of "list" to "tasks" or "taskList", so as to remove ambiguity.